meson.build

```diff
@@ -24,7 +24,7 @@ libd_dep      = dependency('libdaemon',          version : '>=0.14')
 jansson_dep   = dependency('jansson',            version : '>=2.7')
 gst_check_dep = dependency('gstreamer-check-1.0',version : '>=1.0.5')
 thread_dep    = dependency('threads')
-
+libsoup_dep = dependency('libsoup-2.4',		       version : '>=2.4')
```

Comment:
You don't need this.

Reply:
Previous merge (#142) broke meson. To make sure all my changes work I added this.